internal/command/scale/count.go

```diff
@@ -0,0 +1,93 @@
+package scale
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/superfly/flyctl/client"
+	"github.com/superfly/flyctl/internal/appconfig"
+	"github.com/superfly/flyctl/internal/command"
+	"github.com/superfly/flyctl/internal/flag"
+	"github.com/superfly/flyctl/iostreams"
+)
+
+func newScaleCount() *cobra.Command {
+	const (
+		short = "Change an app's VM count to the given value"
+		long  = `Change an app's VM count to the given value.
+
+For pricing, see https://fly.io/docs/about/pricing/`
+	)
+	cmd := command.New("count", short, long, runScaleCount,
```

Comment:
FYI in case this wasn't intentional: previously it also [showed](https://github.com/superfly/flyctl/blob/master/docstrings/gen.go#L640) the argument in the usage. Same thing for `memory` and `vm`.

Reply:
it wasn't but I have to figure how to do it right now :) thanks

Reply:
fixed! 296147e9